---
title: CV
type: page
---

# James Go
---

## Education
- B.S., Cell and Molecular Biology, California State University, Northridge, 2013
- A.S., Administration of Justice, College of the Canyons, 2009

## Research experience
- **Programmer**: University of California, San Diego  
  June 2015 to present
  - Backend and frontend development for a biomedical data website
  - Created REST API services for other researchers and developers to programatically access data
  - Configured servers for testing and production environments and created automatic deployment jobs for continuous integration
- **Research assistant**: University of California, Los Angeles  
  July 2015 to June 2015
  - Created pipeline for analysis of high throughput proteomics data
  - Maintained a Linux webserver and built a web interface for a genome copy number analysis tool
- **Undergraduate fellow**: University of California, Los Angeles  
  August 2013 to July 2014
  - Performed principal component analysis and epigenome-wide association studies on genetic, epigenetic, expression, protein and metabolite data
  - Measured predictive power of linear combinations of CpG methylation sites against quantitative phenotypes
  - Created a web tool to access and visualize associations between phenotypes, genetic markers and epigenetic markers
- **Undergraduate researcher**: California State Universtiy, Northridge  
  June 2011 to July 2013
  - Characterized genes and proteins of interest in nematodes
  - Used chemotaxis assays to study effects of pheromones on different genetic strains
  - Analyzed gene expression under different conditions with fluorescent markers and confocal microscopy

## Skills
- Programming experience building large web applications and tools for data analysis
  - **Languages**: Python, JavaScript, R, PHP, C, HTML, CSS
  - **Tools and frameworks**: React, Angular 1, MySQL, MongoDB, Nginx, Apache, Linux
- Background in biology with experience in both bioinformatics and wet lab work
  - **Wet lab experience**: PCR, Western blotting, chemotaxis assays, cloning
  - **Bioinformatics experience**: Genome and epigenome wide association studies, machine learning on large datasets, RNA-seq analysis, mass spec proteome data analysis

## Publications
- Graham NA, Minasyan A, Lomova A, Cass A, Balanis NG, Friedman M, Chan S, Zhao S, Delgado A, Go J, Beck L, Hurtz C, Ng C, Qiao R, Ten Hoeve J, Palaskas N, Wu H, Müschen M, Multani AS, Port E, Larson SM, Schultz N, Braas D, Christofk HR, Mellinghoff IK, Graeber TG "Recurrent patterns of DNA copy number alterations in tumors reflect metabolic selection pressures". Mol Syst Biol. 2017 Feb 15;13(2):914. doi: 10.15252/msb.20167159.
- Orozco, LD., M. Morselli, L. Rubbi, W. Guo, J. Go, H. Sui, D. Lopez, N.A. Furlotte, B.J. Bennett, C.R. Farber, A. Ghazalpour, M.Q. Zhang, R. Bahous, R. Rozen, A.J. Lusis, M. Pellegrini “Epigenome-wide association of liver methylation patterns and complex metabolic traits in mice”. Cell Metabolism 2015;10.1016/j.cmet.2015.04.025.
- Cinkornpumin, J., R. Wisidagama, V. Rapoport, J. Go, C. Dieterich, X. Wang, R. Sommer, R. Hong “A host beetle pheromone regulates development and behavior in the nematode Pristionchus pacificus". eLife 2014;10.7554/eLife.03229.
